lighthouse-cli/cli-flags.js

```diff
@@ -11,14 +11,25 @@ const yargs = require('yargs');
 const pkg = require('../package.json');
 const printer = require('./printer');
 
+/**
+ * Remove in Node 11 - [].flatMap
+ * @param {Array<Array<string>>} arr
+ * @return {string[]}
+ */
+function flatten(arr) {
+  /** @type {string[]} */
+  const result = [];
+  return result.concat(...arr);
+}
+
 /**
  * @param {string=} manualArgv
  * @return {LH.CliFlags}
  */
 function getFlags(manualArgv) {
   // @ts-ignore yargs() is incorrectly typed as not returning itself
```

Comment:
Maybe update this one as well? `// @ts-ignore yargs() is incorrectly typed as not accepting a single string.`

Reply:
honestly why are we even using `@types/yargs` :)